bool    IK_based_command
uint16  controller_id
float64 x1
float64 x2
float64 x3
